packages/js/src/dashboard/services/widget-factory.js

```diff
@@ -76,16 +80,14 @@ export class WidgetFactory {
 			case WidgetFactory.types.siteKitSetup:
 				// This check here makes sure we don't render the setup anymore if the user connected and then switches away from the dashboard.
 				// Then switches back to the dashboard, but does not refresh.
-				if ( this.#dataProvider.getSiteKitConfiguration().isConnected ||
-				this.#dataProvider.getSiteKitConfiguration().isConfigurationDismissed ) {
+				if ( this.#dataProvider.getSiteKitConfiguration().isConfigurationDismissed ) {
```

Comment:
This is done to prevent hiding of the set up widget once connected so the user can see the completed stepper and dismissed the widget when clicking "Got it".